docker/README.md

```diff
@@ -67,7 +67,14 @@ Jenkins Kubernetes plugin slaves:
 
 ## Build
 
-To build slave execute docker build
+S2I build is required in order to include the 'acpepted license agreement' android sdk
```

Comment:
I would change this to: 
'acpepted license agreement' android sdk =>

We using s2i technology to for building mobile jenkins slaves. During s2i build specific version of the Android SDK is installed and developer is asked to confirm all required licenses.